### ISSUE TITLE
Write an install script

### ISSUE THREAD

**Issue by mdpiper** [csdms/wmt-client#65]
Write an `install` script for **wmt-client** similar to what @mcflugen has done with [csdms/wmt](https://github.com/csdms/wmt) and [csdms/wmt-exe](https://github.com/csdms/wmt-exe).

**Comment by mdpiper** [comment]
The steps that I currently use to build **wmt-client**.

```
$ git clone https://github.com/csdms/wmt-client
$ cd wmt-client
$ ant build
$ # edit war/config.json to set "api_url" and "execution_servers"
$ rm -rf war/data war/save
$ cp -R war /path/to/web/server
```

Consider adding an `ant deploy` command that performs all the tasks after `ant build`. Pass `api_url`, `execution_servers` and the install path as properties with `-D` (see [example](http://stackoverflow.com/a/14309773/1563298)).

**Comment by mdpiper** [comment]
Closed with 97b6b2ec9eca47e966d338bbf8446b8c186d7679.
